### PR TITLE
Test experimental search window

### PR DESCRIPTION
**Pull request by Red-Cat-Fat**
I make SearchWindow as AddComponents:
searchWindiw
But used [SearchWindowContext]( https://docs.unity3d.com/ScriptReference/Experimental.GraphView.SearchWindowContext.html). He mark as experimental:

> Experimental: this API is experimental and might be changed or removed in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,71 @@
...
 
-Support Unity 2019.3, 2020.1 alpha.
+Installation as a unity module via a git link in PackageManager or direct editing of `Packages/manifest' is supported.json:
+```
+"com.elmortem.serializereferenceeditor": "https://github.com/elmortem/serializereferenceeditor.git",
+```
+
+## Main types
+### SRAttribute
+Use it to mark the fields that you want to edit. Important note - they should also be marked with the Serializedreference attribute. You can mark the base type (including the interface) with it.
```
Comment (elmortem) [inline README.md]: Serializedreference -> SerializeReference

File: README.md [hunk 2]
```diff
@@ -1,13 +1,71 @@
...
+[SR]
+public List<AbstractData> DataList = new List<AbstractData>();
+```
+#### Result:
+<img src="Images/Demo.gif" width="400">
+
+#### Additional features
+You can override SRAttribute and implement a rule for processing instantiated objects.
+
+You can see an example in [SRDemoAttribute.cs](serializereferenceeditor%2FDemo%2FSRDemoAttribute.cs), где был переопределён метод `OnCreate`:
```
Comment (elmortem) [inline README.md]: Need translate.

File: README.md (hunk 1 above)
Comment (Red-Cat-Fat) [reply README.md]: Fix

File: README.md (hunk 2 above)
Comment (Red-Cat-Fat) [reply README.md]: Fix

**Pull request merged**